### PR TITLE
Refactor jumpbox role

### PR DESCRIPTION
**Pull request by adborden**
Resolves https://github.com/GSA/datagov-deploy/issues/623

### CODE REVIEW

File: ansible/roles/vendor/gsa.datagov-deploy-jumpbox/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+galaxy_info:
+  author: "GSA"
+  description: "Ansible role to apply configure users on jumpbox"
+  company: GSA
+  license: MIT
```
Comment (JJediny) [inline ansible/roles/vendor/gsa.datagov-deploy-jumpbox/meta/main.yml:5]: Any work we do is CCO Public Domain

**Review by JJediny — APPROVED** [review]
LGTM we just need to change the license in the meta

File: ansible/roles/vendor/gsa.datagov-deploy-jumpbox/meta/main.yml (hunk 1 above)
Comment (adborden) [reply ansible/roles/vendor/gsa.datagov-deploy-jumpbox/meta/main.yml:5]: This is [fixed upstream](https://github.com/GSA/datagov-deploy-jumpbox/pull/1/files#diff-9b4ddf055111f4371f04fbb8b5366655R6), it'll get pulled in on the next version bump.

**Comment by adborden** [comment]
Rebased to resolve conflict.

**Pull request merged**